0-common/scala-core/iglu-core-json4s/src/main/scala/com.snowplowanalytics.iglu.core.json4s/instances.scala

```diff
@@ -0,0 +1,79 @@
+package com.snowplowanalytics.iglu.core
```

Comment:
Add a LICENSE header please here and to all files where it's missing.

Reply:
Done for all files.